internal/test_unit/xml/unit_plot_data_plane.xml

```diff
@@ -292,7 +306,7 @@
     <param> \
       &DATA_DIR_MODEL;/nccf/GloTEC_TEC_2015_03_17.nc \
       &OUTPUT_DIR;/plot_data_plane/plot_data_plane_NCCF_time.ps \
-      'name="TEC"; level="(20150317_000500,*,*)"; file_type=NETCDF_NCCF;' \
+      'name="TEC"; level="(@20150317_000500,*,*)"; file_type=NETCDF_NCCF;' \
```

Comment:
Do we still have tests that ensure that specifying the time value still works without the added @ symbol? If not, I think we should add at least 1 since that syntax is currently used by users.